playground/ty/src/Editor/Editor.tsx

```diff
@@ -311,23 +313,38 @@ class PlaygroundServer
     // If completions is 100, this gives us "99" which has a length of two
     const digitsLength = String(completions.length - 1).length;
 
-    return {
-      suggestions: completions.map((completion, i) => ({
-        label: completion.name,
+    const suggestions: CompletionItem = completions.map((completion, i) => ({
+        label: {
+          label: completion.name,
+          detail: completion.module_name == null
+            ? null
+            : ` (import ${completion.module_name})`,
+          description: completion.detail,
+        },
         sortText: String(i).padStart(digitsLength, "0"),
         kind:
           completion.kind == null
             ? CompletionItemKind.Variable
             : mapCompletionKind(completion.kind),
-        insertText: completion.name,
+        insertText:
+          completion.insert_text == null
+            ? completion.name
+            : completion.insert_text,
```

Comment:
I think you can write this as
```suggestion
          completion.insert_text ?? completion.name
```